client/packages/host/src/api/hooks/utils/useSyncInfo.ts

```diff
@@ -0,0 +1,11 @@
+import { useQuery } from '@openmsupply-client/common';
+import { useHostApi } from './useHostApi';
+
+export const useSyncInfo = (refetchInterval: number | false = false) => {
+  const api = useHostApi();
+
+  return useQuery(api.keys.syncInfo(), api.get.syncInfo, {
+    cacheTime: 0,
+    refetchInterval,
+  });
```

Comment:
ah, if we actually just returned `isSyncing: `data.loading || loading` here it would be quite useful? Seems the use case for this (with the interval etc) is to poll for the status so might always be wanting this 'is syncing' status.. For example adding sync status in the footer with a 30 sec poll, you'd have everything wrapped up nicely

Reply:
Yeah ! Would be awesome to see sync status (last sync was error, last sync too long ago, too many records pending, last sync stuck) etc.. I still getting my head around useQuery thingy, and structure of abstraction, so if i wanted to make a more consumable api hook, I'll do extra data mangling here ?

Reply:
oki, i see sometime we do it in the sdk mapping, like in `getAuthQueries`, hmm, not sure what pattern to use in the future

Reply:
Added a note in https://github.com/openmsupply/open-msupply/issues/688, about documentation for this (it might exists already ?)